org.zenframework.z8.server/src/main/java/org/zenframework/z8/server/apidocs/utils/GsonIntegrator.java

```diff
@@ -0,0 +1,17 @@
+package org.zenframework.z8.server.apidocs.utils;
+
+import com.google.gson.Gson;
```

Comment:
Can we make it without gson?

Reply:
yes we can, but it will be a bit  tricky to visualize readable examples. I just have chosen the easiest way